test/scrabble_test.rb

```diff
@@ -6,8 +6,62 @@
 
 class ScrabbleTest < Minitest::Test
   def test_it_can_score_a_single_letter
-    skip
     assert_equal 1, Scrabble.new.score("a")
     assert_equal 4, Scrabble.new.score("f")
   end
+
+  def test_it_can_score_two_letters
+    assert_equal 2, Scrabble.new.score("as")
+    assert_equal 5, Scrabble.new.score("if")
+  end
+
+  def test_it_can_score_five_letters
+    assert_equal 8, Scrabble.new.score("tasty")
+    assert_equal 8, Scrabble.new.score("waste")
+  end
+
+  def test_returns_zero_score_for_nil_or_empty_input
+    assert_equal 0, Scrabble.new.score(nil)
+    assert_equal 0, Scrabble.new.score("")
+  end
+
+  def nil_or_empty_returns_false_for_word_input
+    refute nil_or_empty('r')
```

Comment:
For this to work, will need to create a new instance of Scrabble, and then call `nil_or_empty` on that instance.

```ruby
def test_nil_or_empty_returns_false_for_word_input
  scrabble = Scrabble.new

  refute scrabble.nil_or_empty('r')
  refute scrabble.nil_or_empty('poop')
end
```